cli/tests/pcluster/cli/configure/test_pcluster_configure/test_enabled_efa_default_placement_group/output.txt

```diff
@@ -0,0 +1,57 @@
+INFO: Configuration file {{ CONFIG_FILE }} will be written.
```

Comment:
Don't we capture and validate the questions/prompts as part of the unit testing? is there a technical limitation?

Reply:
We only capture the output, errors and configs at the moment.

Reply:
is there an easy way to also capture the prompted question? I think that would be extremely valuable to test with the added value of having the entire configure workflow dumped in a file

Reply:
I can take a look at the mechanism used to capture the input to see if it's possible.